react-native-get-random-values.podspec

```diff
@@ -0,0 +1,12 @@
+Pod::Spec.new do |s|
+  s.name         = "react-native-get-random-values"
+  s.version      = "1.1.0"
```

Comment:
Do you think that we could keep this at `0.0.0` all the time, or does it has to be updated to match whichever actual version we are on?

I currently have scripts that bumps, tags, and releases automatically (based on the commit statuses with [emoji-commit](https://github.com/LinusU/emoji-commit)), and would love it if I could keep using that ☺️

Reply:
This can be simplified by reading the current version from the `package.json` file with something like

```
require 'json'

package = JSON.parse(File.read(File.join(__dir__, 'package.json')))

Pod::Spec.new do |s|
  s.name         = package['name']
  s.version      = package['version']
  ...
```

for a full example, see https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec

P.S. Sudden interest in this library comes from https://github.com/brix/crypto-js releasing v4 which now uses `global.crypto.getRandomValues` instead of `Math.random` for the source of randomness.